packages/server/assets/emails/templates/basic/basic.html

```diff
@@ -0,0 +1,566 @@
+      <div
+        style="
+          background: linear-gradient(
+            90deg,
+            rgba(0, 143, 233, 1) 0%,
+            rgba(0, 76, 235, 1) 100%
+          );
+          background-color: linear-gradient(
+            90deg,
+            rgba(0, 143, 233, 1) 0%,
+            rgba(0, 76, 235, 1) 100%
+          );
+          margin: 0px auto;
+          max-width: 600px;
+        "
+      >
+        <table
+          align="center"
+          border="0"
+          cellpadding="0"
+          cellspacing="0"
+          role="presentation"
+          style="
+            background: linear-gradient(
+              90deg,
+              rgba(0, 143, 233, 1) 0%,
+              rgba(0, 76, 235, 1) 100%
+            );
+            background-color: linear-gradient(
+              90deg,
+              rgba(0, 143, 233, 1) 0%,
+              rgba(0, 76, 235, 1) 100%
+            );
+            width: 100%;
+          "
+        >
+          <tbody>
+            <tr>
+              <td
+                style="direction: ltr; font-size: 0px; padding: 0; text-align: center"
+              >
+                <!--[if mso | IE]><table role="presentation" border="0" cellpadding="0" cellspacing="0"><tr><td class="" style="vertical-align:top;width:600px;" ><![endif]-->
+                <div
+                  class="mj-column-per-100 mj-outlook-group-fix"
+                  style="
+                    font-size: 0px;
+                    text-align: left;
+                    direction: ltr;
+                    display: inline-block;
+                    vertical-align: top;
+                    width: 100%;
+                  "
+                >
+                  <table
+                    border="0"
+                    cellpadding="0"
+                    cellspacing="0"
+                    role="presentation"
+                    style="vertical-align: top"
+                    width="100%"
+                  >
+                    <tbody>
+                      <tr>
+                        <td
+                          align="center"
+                          style="
+                            font-size: 0px;
+                            padding: 10px 25px;
+                            word-break: break-word;
+                          "
+                        >
+                          <table
+                            border="0"
+                            cellpadding="0"
+                            cellspacing="0"
+                            role="presentation"
+                            style="border-collapse: collapse; border-spacing: 0px"
+                          >
+                            <tbody>
+                              <tr>
+                                <td style="width: 100px">
+                                  <img
+                                    alt="Speckle"
+                                    height="auto"
+                                    src="https://i.imgur.com/KAvbGEj.png"
```

Comment:
@gjedlicska @didimitrie FYI the logo is currently loaded from imgur, since we don't currently have a way to just dump static assets for public serving anywhere inside `speckle-server`